app/views/mobileValidate.scala.html

```diff
@@ -41,7 +41,12 @@
             <div id="svv-panorama-outline">
                 <div id="svv-panorama">
                     <div id="view-control-layer-mobile" style="z-index: 2">
-                        <div id="label-description-box"></div>
+                        @if("Amsterdam" == cityName){
+                            <div id="label-description-box" class="label-description-box amsterdam"></div>
```

Comment:
Can you change the name of the class from `amsterdam` to sometimes more specific and attached to the behavior like `description-box-visible`?